reana_server/status.py

```diff
@@ -251,11 +262,96 @@ def get_status(self):
         }
 
 
+class QuotaUsageStatus(REANAStatus):
+    """Class to retrieve statistics related to the current REANA users quota usage."""
+
+    def __init__(self, from_=None, until=None, user=None):
+        """Initialise QuotaUsageStatus class.
+
+        :param from_: From which moment in time to collect information. Not
+            implemented yet.
+        :param until: Until which moment in time to collect information. Not
+            implemented yet.
+        :param user: A REANA-DB user model.
+        :type from_: datetime
+        :type until: datetime
+        :type user: reana_db.models.User
+        """
+        super().__init__(from_=from_, until=until, user=user)
+
+    def get_top_five_all(self, resource_type):
+        """Get the top five users according to quota usage."""
+        subq = (
```

Comment:
What do you think of renaming these variables to more explicit names? Like `user_resources_subq ` and `top_five_resource_usage_users`?